package.json

```diff
@@ -75,11 +86,22 @@
     "docusaurus-feed": "./lib/generate-feed.js"
   },
   "devDependencies": {
+    "babel-eslint": "^8.2.5",
+    "eslint": "^5.0.1",
+    "eslint-config-fbjs": "^2.0.1",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-babel": "^5.1.0",
```

Comment:
Are we using them yet? I don't see them in `eslintrc`. Maybe we should add them later if we're not using yet.

Reply:
[eslint-config-fbjs](https://github.com/facebook/fbjs/tree/master/packages/eslint-config-fbjs) requires it as peer dependencies
https://github.com/facebook/fbjs/blob/d308fa83c99c93e8e588de3396cf55b31e56b14e/packages/eslint-config-fbjs/package.json#L21-L28

I've only added `eslint-config-prettier` so far to remove eslint-prettier conflict.